database-plugins/src/main/java/co/cask/hydrator/plugin/db/batch/sink/DBSink.java

```diff
@@ -51,10 +51,7 @@
 import java.sql.ResultSet;
 import java.sql.ResultSetMetaData;
 import java.sql.Statement;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Don't use import *. Please keep the original import (make sure your intelliJ is using the Cask coding style).